cpu/native/include/socket_zep_params.h

```diff
@@ -25,7 +25,7 @@ extern "C" {
 #endif
 
 /**
- * @brief   Number of allocated parameters at @ref socket_zep_params
+ * @brief   Number of allocated parameters at #socket_zep_params
```

Comment:
I don't think this is portable to XML and LaTeX.

Reply:
For me (also doxygen 1.8.15 at the moment) keeping it as it was doesn't pose a problem.

Reply:
Ok, I thought to remember that \ref explicitly requires the definition of an \anchor. However, this only seems to be only the case when you try to refer to a section in another document. I will change it back.

Reply:
With other documents is not correct either. `@anchor` you only need if the thing you want to reference is not auto-anchored (e.g. groups created with `@name`).

Reply:
I just remember the case where I had to add an anchor. I had a document that was defining sections like markdown with `#`.  Then, I had to add the `\anchor` statement explicitly.